README.md: Update ARC-V samples for arc-2025.09 (#666)
foss-for-synopsys-dwc-arc-processors/toolchain: README.md

## Building the Toolchain

> :warning: Note, that `PATH` should not contain toolchains for the same target

> > to prevent builds issue. Also, `LD_LIBRARY_PATH` should be unset too.

Reviewer: ```suggestion
> to prevent builds issue. `LD_LIBRARY_PATH` should be unset as well.
```

Author: Done!
